packages/shared-ux/router/impl/README.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: sharedUX/Router
+slug: /shared-ux/router
+title: Router
+description: A router component
+tags: ['shared-ux', 'component', 'router', 'route']
+date: 2022-08-12
+---
+
+## Summary
+This is a wrapper around the `react-router-dom` Route component that inserts MatchPropagator in every application route. It helps track all route changes and send them to the execution context, later used to enrich APM 'route-change' transactions.
+The component does not require any props and accepts props from the RouteProps interface such as a `path`, or a component like `AppContainer`. 
+
+```jsx
+export interface RouteProps {
+    location?: H.Location;
+    component?: React.ComponentType<RouteComponentProps<any>> | React.ComponentType<any>;
+    render?: (props: RouteComponentProps<any>) => React.ReactNode;
+    children?: ((props: RouteChildrenProps<any>) => React.ReactNode) | React.ReactNode;
+    path?: string | string[];
+    exact?: boolean;
+    sensitive?: boolean;
```

Comment:
Should we add some explanation about what these props are?

Reply:
Yes! Added in a31f7b1 🥳